docs/lib/govuk_tech_docs/open_api/renderer.rb

```diff
@@ -286,7 +286,6 @@ def get_operations(path)
 
       def get_schema_name(text)
         unless text.is_a?(String)
```

Comment:
I think we want this to be something like:

`return unless text.is_a?(String)`

Reply:
Good catch!

Reply:
Updated